plugin/pkg/admission/nodeenv/admission.go

```diff
@@ -0,0 +1,117 @@
+package nodeenv
```

Comment:
Please change the package name to be more appropriate.  Env doesn't really make sense in this context.